karapace/protobuf/schema.py

```diff
@@ -1,14 +1,175 @@
 # Ported from square/wire:
 # wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Schema.kt
 # Ported partially for required functionality.
+from karapace.protobuf.enum_element import EnumElement
+from karapace.protobuf.location import Location
+from karapace.protobuf.message_element import MessageElement
+from karapace.protobuf.option_element import OptionElement
+from karapace.protobuf.proto_file_element import ProtoFileElement
+from karapace.protobuf.proto_parser import ProtoParser
+from karapace.protobuf.utils import append_documentation, append_indented
 
+import logging
 
+log = logging.getLogger(__name__)
+
+
+def static_init(cls):
```

Comment:
This class decorator doesn't seem needed anymore (the value has been changed so that it is defined directly at the class level)

Reply:
ok